openpype/hosts/max/api/plugin.py

```diff
@@ -42,6 +42,10 @@
             (
                 handle_name = node_to_name c
                 node_ref = NodeTransformMonitor node:c
+                idx = finditem list_node.items handle_name
+                if idx do (
+                    return False
+                )
```

Comment:
This doesn't sound right. You wouldn't want to return directly but only continue the for loop, right? So that those in your selection that are **not** already on the list would still get added if they are in the current selection after the already existing ones?

Reply:
At first, i tried with `if idx do continue` it doesn't work. But then I tried your suggestion of` if idx do (continue)` works smoothly. (It's quite strange but at least work). So thank you!